src/Avalonia.Controls/Notifications/SimpleNotification.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Avalonia.Controls.Notifications
+{
+    /// <summary>
+    /// Defines content for a <see cref="Notification"/> control.
+    /// </summary>
+    /// <remarks>
+    /// This notification content type is compatible with native notifications.
+    /// </remarks>
+    public class SimpleNotification : INotification
+    {
+        public static INotification Create (
```

Comment:
Why a static create method instead of a constructor?

Reply:
removed.